objects/obj_item_parentbuff/Create_0.gml

```diff
@@ -0,0 +1,14 @@
+item_name = "";
+item_description = "";
+health = 0 ;
```

Comment:
Can we remove the spacing between the semi-colon to keep things consistent?

Reply:
Sure
Fixed